frontend/src/app/commonComponents/Header.jsx

```diff
@@ -203,7 +208,11 @@ const Header = ({ organizationId }) => {
                   </li>
                   <li className="usa-sidenav__item">{facilityName}</li>
                   <li className="usa-sidenav__item">
-                    <Anchor text={" Log out"} onClick={() => logout()} />
+                    <Anchor
+                      className="prime-force-link-in-nav"
+                      text={" Log out 2"}
```

Comment:
`" log out 2"`? is the `2` intentional

Reply:
D'oh.. Good catch! There were two sections on the page that are almost identical (when the page is made narrow, the vertical menu shows up.) I used this to make double sure I was working on the correct one and forgot to remove it.